src/tests/e2e/api/chef_api_e2e_test.py

```diff
@@ -1,34 +1,58 @@
-from src.constants.http_status_code import HttpStatus
+from src.constants.http import HttpStatus
+from src.constants.oauth2 import Roles
+from src.env_config import get_env_config_instance
 from src.tests.e2e.base_flask_setup_test import BaseFlaskSetupTest
 from src.tests.utils.fixtures.json_entities_fixture import build_chef
 from src.tests.utils.fixtures.mapping_orm_fixtures import create_order_with_procedure
+from src.tests.utils.fixtures.oauth2_fixture import build_client, build_user
+from src.utils.oauth2_util import build_user_credential_access_token
 
 
 class ChefApiE2ETest(BaseFlaskSetupTest):
     def after_base_setup(self):
-        pass
+        env_config = get_env_config_instance()
+        user = build_user(roles=Roles.ADMINISTRATOR.value)
```

Comment:
Don't you need to do the same thing for next endpoints? can you put this into an test util file? maybe an access token fixture?

Think a better way to reuse this

Reply:
Done. I created a `token_fixture` file to create all the different tokens for test.